jenkins-design-language/src/js/components/pipeline/PipelineGraphLayout.ts

```diff
@@ -163,6 +163,9 @@ function positionNodes(nodeColumns: Array<NodeColumn>, { nodeSpacingH, parallelS
             xp += Math.round((widestRow - row.length) * parallelSpacingH * 0.5);
 
             for (const node of row) {
+                if (!node.isPlaceholder && node.stage && node.stage.seqContainerName) {
+                    xp += 70;
```

Comment:
We probably want to put this as an `export const` so that it has a name, and use that value (÷2) when offsetting the top label position in PipelineGraph

Reply:
Will do, thanks

Reply:
Done :)